website/django/data/migrations/0042_variantrepresentation.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2019-06-18 16:05
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data', '0041_paper_crawl_date'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='VariantRepresentation',
+            fields=[
+                ('Variant', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='data.Variant')),
```

Comment:
Curious if it should be a one-to-one relationship or if we just want all variants with a given genomic coordinate to point to the same VariantRepresentation. Should probably get @melissacline's input on this, but I understand the only reason to do one-to-one is to allow versioning to take VR representation into account -- is that correct @falquaddoomi ?

Reply:
From Melissa on Slack:

> Yes, the VR representation is just static data associated with the genomic coordinate.  This “static” data will evolve as the VR representation evolves.  I’m not picturing any use case for old versions of the VR representation.  So, I’d say that it doesn’t need to be versioned.